unity-renderer/Assets/DCLPlugins/FallbackAssetLoaderPlugin/FallbackFontsLoaderPlugin.cs

```diff
@@ -0,0 +1,34 @@
+using DCL;
+using System.Collections.Generic;
+using TMPro;
+
+namespace DCLPlugins.FallbackFontsLoader
+{
+    public class FallbackFontsLoaderPlugin : IPlugin
+    {
+        private readonly string FALLBACK_FONTS_ADDRESS = "Assets/Fonts/Fonts Sources/";
+        private readonly List<string> FALLBACK_FONTS_TO_LOAD = new () { "NotoSansJP-SemiBold SDF.asset", "NotoSansSC-SemiBold SDF.asset", "NotoSansKR-SemiBold SDF.asset" };
+
+        public FallbackFontsLoaderPlugin()
+        {
+            foreach (string font in FALLBACK_FONTS_TO_LOAD)
+            {
+                var additionalFonts = new AssetPromise_Font(FALLBACK_FONTS_ADDRESS + font);
+                additionalFonts.OnSuccessEvent += AddFallbackFont;
+                AssetPromiseKeeper_Font.i.Keep(additionalFonts);
+            }
+        }
+
+        public void Dispose() { }
```

Comment:
Does it make sense to add promises forgetting on `Dispose`?

Reply:
Yeah, I think it does. Adding it